stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ChatClient.kt

```diff
@@ -161,6 +163,7 @@ public class ChatClient internal constructor(
         PushNotificationReceivedListener { _, _ -> }
 
     public lateinit var plugins: List<Plugin>
+    public lateinit var errorHandlers: List<ErrorHandler>
```

Comment:
We can have multiple error handlers, just like plugins. This way, if an error handler fails, more fallbacks can be used.